tls/vibe/stream/openssl.d

```diff
@@ -49,6 +49,10 @@ version(VibeForceALPN) enum alpn_forced = true;
 else enum alpn_forced = false;
 enum haveALPN = OPENSSL_VERSION_NUMBER >= 0x10200000 || alpn_forced;
 
+version(VibeForceKeylog) enum keylog_forced = true;
```

Comment:
I would remove this `VibeForceKeylog` and instead twek it so that `VibeKeylogFromEnvironment` is taken into account. This way you don't have to have two checks on L802 (in the ctor).

Reply:
Or just remove this part and have the `version` in ctor

Reply:
I removed the keylog_forced thing, but still have the version for the env var reading.